training.py

```diff
@@ -0,0 +1,75 @@
+import argbind
+from datetime import datetime
+from pathlib import Path
+import time
+from torch.utils.data import DataLoader
+import torch.optim as optim
+import torch
+from torch.utils.tensorboard import SummaryWriter
+
+import data
+from model import Scribe
+import utils
+
+@argbind.bind(without_prefix=True)
+def train(
+        save_path: Path = Path('runs') / datetime.now().isoformat(),
+        num_training_iterations: int = 1000,
+        batch_size: int = 40,
+        random_seed: int = None):
+
+    """Trains model, samples an output, and displays it
+
+    Parameters
+    ----------
+    save_path : Path
+        The location of the tensorboard run (defaults to runs/CURDATETIME)
+    """    
+    writer = SummaryWriter(save_path)
+    if random_seed:
+        print(f"Setting random seed: {random_seed}")
+        torch.manual_seed(random_seed)
+    dataset = data.HandwritingDataset()
+    dl = DataLoader(dataset, shuffle=True, batch_size=batch_size, collate_fn=data.collate_fn)
+    inf = utils.infinite_dl(dl)
```

Comment:
I'd just do: `dl = utils.infinite_dl(dl)`. It would also prevent bugs as the one in the for loop.